src/Azure.Functions.Cli/StaticResources/funcignore

```diff
@@ -5,3 +5,8 @@
 local.settings.json
 test
 getting_started.md
+node_modules/.bin/
+node_modules/@azure/
```

Comment:
We can't exclude this one because the `@azure` scope has a lot of sdks and whatnot that people use in production.

The "@azure/functions" package is also shifting to a production dependency (More info [here](https://github.com/Azure/azure-functions-nodejs-worker/issues/568) if you're curious).